cmds/live.c

```diff
@@ -240,6 +240,13 @@ static int forward_options(struct uftrace_opts *opts)
 		goto socket_error;
 	}
 
+	if (opts->trace != TRACE_STATE_NONE) {
+		bool trace = opts->trace == TRACE_STATE_ON;
+		if ((ret = forward_option(sfd, AGENT_TRACE, &trace, sizeof(bool))) < 0)
```

Comment:
Please break the statement and don't assign a variable in the condition.  It's just a matter of preference and I like simple things. :)

And for I/O, I think we'd better using a sized int (like uint8_t or int32_t) rather than bool.